code-challenges.test.js

```diff
@@ -24,25 +29,90 @@
 
 // b) Create the function that makes the test pass.
 
+//create fib variable
+//for fib define num values 
+//for loop states that the function will act on numbers greater than 2, but less than or equal to the example inputs (6,10). Function will iterate through the given parameters
+//return array of 6 and 10 in fib.
+
+
+
+
+var i;
+var fib = [];
+
+fib[0] = 0;
+fib[1] = 1;
+for (i = 2; i <= 10; i++) {
+  fib[i] = fib[i - 2] + fib[i - 1];
+  console.log(fib[i]);
+}
+--------------------------------------
+var i;
+var fib = [];
+
+fib[0] = 0;
+fib[1] = 1;
+for (i = 2; i <= 6; i++) {
+  fib[i] = fib[i - 2] + fib[i - 1];
+  console.log(fib[i]);
+}
 
```

Comment:
Here is an extended explanation that I gave another developer on the fibonacci sequence. Please let me know if this helps.
 Pseudocode
 function name fib
 input/parameter is a number
 output is an array
 Research of fibonacci sequence...if the array with a length of 6 is equal to [1, 1, 2, 3, 5, 8], so starting at index2, we add the previous indexes to create the value for the following index that means the value for each index is as follows:
index0 = 1
index1 = 1
index2 = index0 + index1 = 1 + 1 = 2
index3 = index1 + index2 = 1 + 2 = 3
index4 = index2 + index3 = 2 + 3 = 5
index5 = index3 + index4 = 3 + 5 = 8
    
 Start with an array [1, 1]
 push the values of the indexes into  that will be added to the array

declare a function, told it to look at the parameter which is a number
 `const fib = (num) => {`

   // start with an array of [1, 1], declare a variable to hold that array
   `let seq = [1, 1]`

   // iterated with a for loop starting at 0; 
   // the length has to be reconsidered because our array begins with a length of 2 already for the fibonacci sequence [1, 1]. Therefore, we subtract 2 from the number so we can received the desired length;
   // and next we tell it to increment up
`   for(let i = 0; i < num - 2; i++) {`

pushing the next value of fibonacci sequence to the seq array, used an equation based on the fibonacci research that adds the previous two indexes
`     seq.push(seq[i] + seq[i+1])
   }`

   //return the mutated array
 `  return seq 
 }`

 This illustrates how the for loop does it iteration on the seq array when passed the number 6.
 1st iteration
 [1, 1, (seq[0] + seq[0+1])]
 [1, 1, (1 + 1)]
 [1, 1, 2]

 2nd iteration
 [1, 1, 2, (seq[1] + seq[1+1])]
 [1, 1, 2, (1 + 2)]
 [1, 1, 2, 3]

 3rd iteration
 [1, 1, 2, 3, (seq[2] + seq[2+1])]
 [1, 1, 2, 3, (2 + 3)]
 [1, 1, 2, 3, 5]

 4th iteration
 [1, 1, 2, 3, 5, (seq[3] + seq[3+1])]
 [1, 1, 2, 3, 5, (3 + 5)]
 [1, 1, 2, 3, 5, 8]

 Here's the function without any pseudocode:

 `const fib = (num) => {
   let seq = [1, 1]
   for(let i = 0; i < num - 2; i++) {
     seq.push(seq[i] + seq[i+1])
   }
   return seq 
 }`